src/Process.text

```diff
@@ -0,0 +1,76 @@
+BREAKING DOWN THE PROCESS
+
+
+1 - Assign a value to each letters ✅
+#Prima devo assegnare ad ogni lettera un valore ✅
+--> Create an object with letters and their values ✅
+#Creare un oggetto con lettere e conseguente valore ✅
+
```

Comment:
very detailed breakdown of the process made to achieve your result. Amazing work😁